src/main/java/org/jvnet/hudson/reactor/ReactorListener.java

```diff
@@ -38,14 +38,18 @@
     /**
      * Notifies that the execution of the task is about to start.
      */
-    void onTaskStarted(Task t);
+    default void onTaskStarted(Task t) {
+        // Do nothing by default
```

Comment:
Is there any risk of developers forgetting to override these now? From what I can tell the class is only used when you execute a reactor, and the existing usages are all in Jenkins core, so I don't think there is any issue, but wanted to make sure.

Reply:
Well, it is a listener. I think it should not be a problem in this interface, because subscribes just listen for events. A developer just needs to read the documentation. The lib is not used outside Jenkins AFAICT, so it should not be a problem anyway